exercises/exercise-05/index.ts

```diff
@@ -78,6 +85,11 @@ function filterPersons(persons: Person[], personType: string, criteria: unknown)
 let usersOfAge23: User[] = filterPersons(persons, 'user', { age: 23 });
 let adminsOfAge23: Admin[] = filterPersons(persons, 'admin', { age: 23 });
 
+// @ts-ignore
```

Comment:
This would be a great use of `@ts-expect-error`: filtering to admins shouldn't allow you to select by occupation.